docs/_components/headers/headers.html

```diff
@@ -3,82 +3,158 @@
 title: Headers
 category: UI components
 permalink: /headers/
-lead: Headers introduce a website and provide a quick, organized way to reach the main sections of a website.
+lead: 'Headers help users identify where they are and provide a quick, organized way to reach the main sections of a website. Our headers include an “official website of the U.S. government” banner, branding to identify the site, and horizontal navigation.'
 maturity: alpha
 ---
 
-<!-- <h3 class="usa-heading heading-margin-alt">Thematic header documentation</h3> -->
+<h3 class="usa-heading heading-margin-alt">Accessibility</h3>
+<ul class="usa-content-list">
+  <li>Include skip navigation links to allow users with screen readers to bypass long navigation lists. Make sure you include an id at the beginning of your main content and that it matches the skipnav link. Find more information here: <a href="http://webaim.org/techniques/skipnav/">http://webaim.org/techniques/skipnav/</a></li>
+  <li>Include Tab focus for all top-level navigation navigation items — this will allow keyboard-reliant users to easily navigate interactive items.</li>
+  <li>Ensure your horizontal navigation is keyboard compatible; test to make sure users can use Tab to navigate and Space (or Enter) to open pages.</li>
+  <li>Avoid using hover to expand dropdown lists. Hover is difficult for some users and won't work on touch screens. Drop downs should expand on click or with keyboard navigation.</li>
+  <li>We use semantic header and nav elements with <code>role="banner"</code> and <code>role="navigation"</code> respectively. <code>role="banner"</code> is your masthead.</li>
+  <li>You can use multiple nav elements for groups of navigation links, but you should only use one <code>role=”navigation”</code> for the main nav of a page.</li>
```

Comment:
The quotes around `"role=navigation"` are a different typeset than the others. Might be a copy/paste error.